### DISCUSSION TITLE
ComfyUI theming

### DISCUSSION THREAD

**Discussion by CasanovaSan** [Comfy-Org/ComfyUI#5948] [Ideas]
I wish that more people would start working on creating themes for comfyui

Currently i only know of two and that's the Kitchen theme  which hasn't been getting any update and i am kinda sad about that, it looks really good and could've been such a good ui
![screenshot-1](https://github.com/user-attachments/assets/f5efd1e3-e5f5-4a40-aa1d-f6bf9da99f38)
https://github.com/canisminor1990/kitchen-comfyui

And SwarmUI

are there any other themes that i don't know of?

**Comment by wibur0620** [comment]
Comfyui now has its own front end

**Reply by doggeddalle** [reply]
> Comfyui now has its own front end

That wasn't the question. And the frontend is currently trash.
